src/components/IconButton/IconButton.driver.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { StylableUnidriverUtil, UniDriver } from 'wix-ui-test-utils/unidriver';
+import style from './IconButton.st.css';
+
+export interface IconButtonDriver extends BaseUniDriver {
+  isMobile(): Promise<boolean>;
```

Comment:
From what I see in the spec, there is no `mobile` design for this component.
So you can remove all the references to `mobile` that were generated automatically